themes/scss/widgets/icon-modifiers.scss

```diff
@@ -0,0 +1,12 @@
+// Icon fonts modifiers
+// ----------------------------------------------
+
+.CDB-IconFont--center {
+  vertical-align: middle;
+}
+.CDB-IconFont--top {
+  vertical-align: top;
+}
+.CDB-IconFont--small {
+  font-size: $sFontSize-small + 1;
```

Comment:
Name of variable `sFontSize-small` should be written in all lowercase letters with hyphens instead of underscores